library/Booster/Syntax/Json/Externalise.hs

```diff
@@ -88,14 +88,14 @@ externalisePredicate sort =
                 Syntax.KJExists
                     { sort
                     , var = varNameToId var.variableName
-                    , varSort = externaliseSort $ var.variableSort
+                    , varSort = externaliseSort var.variableSort
                     , arg = recursion p1
                     }
             Internal.Forall var p1 ->
                 Syntax.KJForall
                     { sort
                     , var = varNameToId var.variableName
-                    , varSort = externaliseSort $ var.variableSort
+                    , varSort = externaliseSort var.variableSort
```

Comment:
Really saving a lot of dollars :moneybag: , great !!!  :smirk:

Reply:
It's `hlint`, probably we should forward the dollars to them!